awesomeConfig.cmake

```diff
@@ -332,6 +334,7 @@ if(DO_COVERAGE)
                     ${BUILD_DIR}/${file}
                     COPYONLY)
     endforeach()
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -O0 --coverage -fprofile-arcs -ftest-coverage")
```

Comment:
Shouldn't the commit also remove the part of `.travis.yml` which added these flags? Also, why are these different flags than what `.travis.yml` set? What happened to `$LDFLAGS`?